README.md

```diff
@@ -1,4 +1,4 @@
-# Helm Chart for Harbor
+****# Helm Chart for Harbor
```

Comment:
@sylvainOL , could you please justify why we need change of this line? Could you please remove the change regarding this line, thanks